hw_06/task_02.js

```diff
@@ -0,0 +1,100 @@
+// 1. У вас есть массив названий пицц вашего конкурента. Создайте функцию, которая будет принимать ваш набор названий пицц (массив) 
+//   и возвращать только те, которых нет у конкурента (тоже массив). Если все ваши пиццы есть у конкурента - вернуть null
+
+
+const competitorPizzas = ['Pepperoni', 'Capricciosa', 'Diavola', '4 formaggi', 'Hawai'];
+
+function comparePizzas (arrayPizzas) {
+    
+    const comparasionResult = [];
+
+    for (let i = 0; i < arrayPizzas.length; i++) {
+           if (!competitorPizzas.includes(arrayPizzas[i])) {
```

Comment:
Suggestion:
Convert all pizzas to lower case before comparing to avoid case bugs